### PR TITLE
fix for unknown ssize_t in aten/src/TH/THMemoryFile.c

### PR DESCRIPTION
**Pull request by andreh7**
added `sys/types.h` include to fix unknown `ssize_t` in `aten/src/TH/THMemoryFile.c` with gcc 4.8.5 on Linux

(as reported in #3611)

### CODE REVIEW

File: aten/src/TH/THMemoryFile.c [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #include "THMemoryFile.h"
 #include "THFilePrivate.h"
 #include "stdint.h"
+#include <sys/types.h>
```
Comment (apaszke) [inline aten/src/TH/THMemoryFile.c]: @peterjc123 this has to be wrapped in `#ifndef _WIN32`, right?

Comment (peterjc123) [reply aten/src/TH/THMemoryFile.c]: yes, i think so.

**Comment by soumith** [comment]
thanks @andreh7

**Pull request merged**